subprojects/antlr/src/main/java/org/gradle/api/plugins/antlr/AntlrTask.java

```diff
@@ -271,16 +281,27 @@ public void setSource(Object source) {
     }
 
     /**
-     * Returns the source for this task, after the include and exclude patterns have been applied. Ignores source files which do not exist.
-     *
-     * @return The source.
+     * {@inheritDoc}
      */
     @Override
-    @PathSensitive(PathSensitivity.RELATIVE)
+    @Internal
```

Comment:
Should we maybe use `@ReplacedBy` instead of `@Internal`?

Reply:
Sounds better to me, too.

Reply:
Using `@ReplacedBy` now.

Reply:
Looks like we can't use `@ReplacedBy` for now, since property validation will fail:

```
Task property validation failed. See https://docs.gradle.org/6.0-20190823180744+0000/userguide/more_about_tasks.html#sec:task_input_output_annotations for more information on how to annotate task properties.
> Warning: Type 'org.gradle.api.plugins.antlr.AntlrTask': property 'source' is annotated with @PathSensitive that is not allowed for @ReplacedBy properties
```

Let's keep internal and fix property validation for `@ReplacedBy` in a separate PR.